packages/xo-web/src/common/select-objects.js

```diff
@@ -250,24 +260,28 @@ class GenericSelect extends React.Component {
   }
 
   // GroupBy: Display option with margin if not disabled and containers exists.
-  _renderOption = option => (
-    <span
-      className={
-        !option.disabled && this.props.xoContainers !== undefined
-          ? 'ml-1'
-          : undefined
-      }
-    >
-      {renderXoItem(option.xoItem, {
-        type:
-          this.props.resourceSet !== undefined &&
-          option.xoItem.type !== undefined
-            ? `${option.xoItem.type}-resourceSet`
-            : undefined,
-        memoryFree: option.xoItem.type === 'host' || undefined,
-      })}
-    </span>
-  )
+  _renderOption = option => {
+    const isHost = option.xoItem.type === 'host'
+    return (
+      <span
+        className={
+          !option.disabled && this.props.xoContainers !== undefined
+            ? 'ml-1'
+            : undefined
+        }
+      >
+        {renderXoItem(option.xoItem, {
+          type:
+            this.props.resourceSet !== undefined &&
+            option.xoItem.type !== undefined
+              ? `${option.xoItem.type}-resourceSet`
+              : undefined,
+          memoryFree: isHost || undefined,
+          pool: isHost ? false : undefined,
```

Comment:
Is this change related to the PR?

Reply:
Yes, for not having duplicate information (host's pool).

Reply:
What about when you search a host by typing in the select. You need to see the pool's name.